package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "content-service",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I still recommend `1.1.0` as this is a new functionality that doesn't break anything versus a bug patch that doesn't break anything. See https://semver.org/#summary for more info.